src/public/parts/shells/chat/public/settings/settings.css

```
@@ -48,7 +48,7 @@
 .settings-nav-group {
 	margin-top: 0.55rem;
 	padding-top: 0.55rem;
-	border-top: 1px solid color-mix(in srgb, currentColor 10%, transparent);
+	border-top: var(--border) solid color-mix(in srgb, currentColor 10%, transparent);
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)